helix-term/src/ui/menu.rs

```diff
@@ -394,19 +394,6 @@ impl<T: Item + 'static> Component for Menu<T> {
 
         let render_borders = cx.editor.menu_border();
 
-        if !render_borders {
```

Comment:
May be missing something but why was this removed?

Reply:
sorry it was a mistake. (this why it was in draft, because it was late)